### ISSUE TITLE
When rebuilt, carousel doesn't work

### ISSUE THREAD

**Issue by orangecms** [brainhubeu/react-carousel#72]
**Describe the bug**
When I rebuild the lib (`b292ec4849eea5c9c95fe658e35d9a8491434dea`) and add it to my website, the slider doesn't move.

**To Reproduce**
`yarn run build` and add it to your website

**Expected behavior**
The carousel should move

**Screenshots**
:woman_shrugging: 

**Desktop (please complete the following information):**
any

**Smartphone (please complete the following information):**
any

**Additional context**
When I revert `91f80fa5d2ea11c915bdbadffff50cb7bf859067`, it works again.

**Comment by mdrazyk** [comment]
Hey @orangecms
Can you just install this dependency from npm as described in README? I think that would be easier :)

**Comment by orangecms** [comment]
Sure, I did that, but I ran into #47, so I worked on #74, but I had to rebuild to try it out, and then the carousel stopped moving. So I went back to master, rebuilt, and it still didn't move. I got different build artifacts as well, i.e., git status told me that `lib/` had changed, i.e., I couldn't reproduce it. Then I reverted other patches to see where it broke, and that seemed to be the commit I mentioned up there.
